src/pages/instances/InstanceDetail.tsx

```diff
@@ -76,10 +76,23 @@ const InstanceDetail: FC = () => {
               className="p-panel__title"
               inline
               items={[
-                <span key="name">{name}</span>,
```

Comment:
I think we don't need this list as a wrapper. Removing it should also simplify the styling. I.e. the magic 0.4 in `p-breadcrumbs__item margin-bottom: 0.4rem;` should be a 0. Because the flex container with `align-items: center` should do the centring. The problem with the list is, that the items are hidden away under the list and don't get centred, when they differ in height. I hope you understand what I meant.

Reply:
I quickly tried to remove the list and it was causing only additional problems. If you have something specific in mind that you tested and it's working, feel free to submit a suggestion with the suggestion button:

![image](https://user-images.githubusercontent.com/56583786/228616618-5ab3116e-be75-4c40-b1f6-7bf6b2ac177b.png)

Reply:
This is what I came up with. Quite a few styles still needed, but at least the list is gone:
```
            <div className="p-panel__title">
              <nav
                key="breadcrumbs"
                className="p-breadcrumbs"
                aria-label="Breadcrumbs"
              >
                <ol className="p-breadcrumbs__items">
                  <li className="p-breadcrumbs__item">
                    <Link to={`/ui/${project}/instances`}>Instances</Link>
                  </li>
                  <li className="p-breadcrumbs__item">
                    <span>
                      {name}{" "}
                      <i className="u-text--muted">({instance.status})</i>
                    </span>
                  </li>
                </ol>
              </nav>
              <InstanceStateActions key="state" instance={instance} />
            </div>
```
and the styles will become

```

  .p-panel__title {
    align-items: center;
    display: flex;
    padding-bottom: $spv--x-small;

    .p-breadcrumbs {
      margin-bottom: 0.2rem;
      width: inherit;

      .p-breadcrumbs__items {
        margin-bottom: 0;
      }

      .p-breadcrumbs__item {
        margin-top: 0;
      }

      .p-breadcrumbs__item:first-of-type {
        margin-right: $sp-x-small;
      }
    }

    .actions-list {
      margin-bottom: 0;
      margin-left: $sph--large;
    }
  }
```

Reply:
This is super helpful, thanks! I also came up with a similar solution eventually, but I was struggling with the alignment on smaller screens. This has a bit more styles than my solution, but it also works nicer at the different resolutions.

Reply:
Maybe the actions need a `min-width` in general, to avoid them from breaking in two lines. We'd have to check if that works well in the instance list and here in the header. Might be good for both cases.

Reply:
Ok, I added also the `min-width`, but they're still breaking if you zoom enough - I don't know if there's much we can do about it.